src/components/tooltip/tooltip.js

```diff
@@ -36,15 +35,9 @@ export const BTooltip = /*#__PURE__*/ Vue.extend({
       type: String,
       default: 'top'
     },
-    fallbackPlacement: {
-      type: [String, Array],
-      default: 'flip',
-      validator(value) {
-        return (
-          (isArray(value) && value.every(v => isString(v))) ||
-          arrayIncludes(['flip', 'clockwise', 'counterclockwise'], value)
-        )
-      }
+    fallbackPlacements: {
+      type: Array,
+      default: undefined
```

Comment:
This would be considered a breaking change not supporting a single string

Reply:
Thats true. We should add back the `fallbackPlacement` prop an deprecate it.